cockpit/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM node:11-alpine as builder
+
+WORKDIR /cockpit
+COPY package*.json ./
+RUN npm install
+COPY . .
+RUN npm run compile
+
+FROM node:11-alpine
+RUN apk --no-cache upgrade
+
+COPY --from=builder /cockpit/dist /cockpit/dist
+WORKDIR /cockpit
+
+COPY package*.json ./
+
+ENV NODE_ENV=production
+RUN npm install
+EXPOSE 3000

```

Comment:
Can we please use port 10000 by default like in all other modules

Reply:
but since the cockpit needs to always be used with the app-connector client shouldn't it be different than it, as default, since standalone they should be different?

Reply:
hmm.. yes